app/controllers/events_controller.rb

```diff
@@ -4,6 +4,15 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
+    @breadcrumbs = [
+      { label: 'events', url: events_path },
+      { label: @event.name, url: event_path(@event) },
+      { label: 'participants' }
+    ]
+    @users = @event.users.travel
```

Comment:
Actually, accessible_by checks if the current ability can fetch that record and if we use it here https://github.com/openSUSE/travel-support-program/pull/80/files#diff-3a4e95c1bc41d3110d2f9ac10064e79bR13 it will not be able to fetch any records because tsp members have no extra permission to view a User record.

A tsp member can only read the User Profile of a user https://github.com/openSUSE/travel-support-program/blob/master/app/models/ability.rb#L137

So we can define, can :read, Request, type: "TravelSponsorship" in the tsp role and then fetch the requests using accessible_by and then fetch the users from it, but I think it is better to use users only with the requests type as TravelSponsorship as a tsp member can view all these requests.